src/tk/shanebee/lesspillagers/LessPillagers.java

```diff
@@ -4,66 +4,100 @@
 import org.bukkit.ChatColor;
 import org.bukkit.command.Command;
 import org.bukkit.command.CommandSender;
+import org.bukkit.configuration.ConfigurationSection;
 import org.bukkit.entity.Entity;
 import org.bukkit.entity.EntityType;
 import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
+import org.bukkit.event.HandlerList;
 import org.bukkit.event.Listener;
 import org.bukkit.event.entity.CreatureSpawnEvent;
 import org.bukkit.plugin.java.JavaPlugin;
 
-import java.util.List;
-
 public class LessPillagers extends JavaPlugin implements Listener {
+  private int MAX_PILLAGERS, RADIUS_X, RADIUS_Y, RADIUS_Z;
```

Comment:
The only issue I see is that this seems to be indented with 2 spaces vs the standard 4. Plus I have my IDE set to use tabs (I need to get into the habit of 4 spaces), but if you could change this to either 4 spaces or tabs that would be appreciated.